MantisPhpClient.php

```diff
@@ -347,15 +363,16 @@ private function getProjectFromProjectTree( $p_project_id, $p_projects ) {
     /**
      * Checks if a user exists.
      *
-     * @param $p_user array that can contain id, name, or email.
+     * @param StdClass $p_user array that can contain id, name, or email.
      * @return true found, false otherwise.
+     * @throws Exception
```

Comment:
Clarify when we throw an exception, e.g. when used against an old version of MantisBT.